src/mixer.c

```diff
@@ -760,6 +760,51 @@ struct mixer_ctl *mixer_get_ctl_by_name_and_index(struct mixer *mixer,
     return NULL;
 }
 
+/** Gets an instance of mixer control handle, by the mixer control's name and device.
+ *  For instance, if two controls have same name, e.g. 'Playback Channel map', then PCM device returns the specific control.
```

Comment:
For better readability, the column limit is 100 chars. Please help to make sure all lines don't exceed.

Reply:
updated